subprojects/shipkit/src/main/groovy/org/shipkit/internal/notes/util/Supplier.java

```diff
@@ -0,0 +1,6 @@
+package org.shipkit.internal.notes.util;
+
+public interface Supplier<T> {
```

Comment:
Can you add a TODO to remove when we migrate to Java8?

I like this new interface. I would introduce it myself :)